### DISCUSSION TITLE
Perl (legacy?) Code

### DISCUSSION THREAD

**Discussion by RPINerd** [ablab/spades#1088] [Ideas]
So I hope this isn't too far off from "Ideas", but long story short I needed to dive a bit into the spades repo today and noticed that theres a handful of perl scripts that seem just.. idle.

I cloned the repo and did some quick searching on a few and found no references or calls to them. As a random example, qualfa2fq at /assembler/ext/src/bwa/qualfa2fq.pl is only ever referred to in an old change log as a git add file.

Would it be welcomed if I went through and removed any orphan scripts I find and submit a pull for review? Or are these present for some other reason that is not obvious from the code itself?

**Comment by asl** [comment]
Usually they are from 3-rd party code. We'd like to keep them in tree in order to resolve the differences easier should we need to update the version of such dependency
